### PR TITLE
Feat: Option to minimize the main window at startup

### PR DESCRIPTION
**Pull request by hzdrro**
#1097

### CODE REVIEW

**Review by DuckSoft — APPROVED** [review]
LGTM

File: translations/zh_CN.ts [hunk 1]
```diff
@@ -1938,6 +1938,10 @@ Output: <byte value="xd"/>
       <source>Test Latency On Connected</source>
       <translation>连接时测试延迟</translation>
     </message>
+    <message>
+      <source>Start Minimized</source>
+      <translation>启动时最小化</translation>
+    </message>
```
Comment (U-v-U) [inline translations/zh_CN.ts:1944]: Please revert the changes in translation target files since they are handled by Crowdin

**Review by U-v-U — CHANGES_REQUESTED** [review]

File: translations/zh_CN.ts (hunk 1 above)
Comment (DuckSoft) [reply translations/zh_CN.ts:1944]: I will later do a manual sync

**Comment by U-v-U** [comment]
Build version is not updated.
Translations should not be committed here.

**Comment by DuckSoft** [comment]
I will bump the build version.
